better_bookmarks/js/content_script.js

```diff
@@ -0,0 +1,17 @@
+const title = document.getElementsByTagName("title")[0];
+const metas = document.getElementsByTagName("meta");
+const url = document.URL;
+var description;
+for (let i = 0; i < metas.length; i++) {
+    const meta = metas[i];
+    // Some websites don't conform to the norm, thus requiring toLowerCase()
+    if (meta.name.toLowerCase() == "description"){
+        description = meta.content;
+    }
+}
+if(description === undefined){
```

Comment:
If you can't actually write something like, ```chrome.storage.sync.set({title: title, desc: description})```, then rewrite all this to be the following:

```javascript
if (!description) {
    description = "";
}
chrome.storage.sync.set({title: title.innerText, desc: description, url: url});
```

This is a cleaner way to write this if you can't do the one liner in the first sentence.  If you find yourself repeating yourself in code, immediately ask yourself why you are finding it necessary to repeat yourself and think of a solution that avoids unnecessary repetition.